providers/aws/resources/aws.lr

```diff
@@ -1822,11 +1822,13 @@ private aws.s3.bucket.corsrule @defaults("name") {
 }
 
 // Amazon S3 bucket policy
-private aws.s3.bucket.policy @defaults("name version") {
+private aws.s3.bucket.policy @defaults("bucketName version") {
   // Unique ID for the policy
   id string
-  // Name for the policy
+  // Deprecated
```

Comment:
nit: can you add "use bucketName instead" so that the deprecation reason is clear?

Reply:
Done